### PR TITLE
Add logos - issue #102

### PR DESCRIPTION
**Pull request by mlocardpaulet**

### CODE REVIEW

File: webinterface/_base.py [hunk 1]
```diff
@@ -35,15 +35,24 @@ def _preface(self):
             """,
             unsafe_allow_html=True,
         )
+        st.image("https://github.com/Proteobench/ProteoBench/blob/Add-logos/webinterface/logos/logo_participants/logos_all_20230926.png?raw=true")
```
Comment (enryH) [inline webinterface/_base.py:38]: I guess you discussed this, but what is the reason to use the github url? Wouldn't the application be shipped with the images and the relative path be enough?

File: webinterface/logos/logo_proteobench/source material/MuseoModerno Regular.ttf
Comment (enryH) [inline webinterface/logos/logo_proteobench/source material/MuseoModerno Regular.ttf:1]: filepath with space? does ttf format work?

File: webinterface/_base.py (hunk 1 above)
Comment (mlocardpaulet) [reply webinterface/_base.py:38]: We did not discuss this. I just did it this way for lack of knowing better I think. 
I am totally open to suggestions :)

File: webinterface/pages/DDA_Quant.py [hunk 2]
```diff
@@ -110,12 +110,12 @@ def _main_page(self):
...
                     Q-Exactive H-FX (ThermoScientific). They can be downloaded from the
-                    proteomeXchange repository PXD028735. You can download them here:
-                    [LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_01.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_01.raw)
-                    [LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_02.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_02.raw)
-                    [LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_03.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_03.raw)
-                    [LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_01.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_01.raw)
-                    [LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_02.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_02.raw)
+                    proteomeXchange repository PXD028735. You can download them here:
```
Comment (enryH) [inline webinterface/pages/DDA_Quant.py:113]:
```suggestion
                    proteomeXchange repository PXD028735. You can download them here:
```

Comment (enryH) [reply webinterface/pages/DDA_Quant.py:113]: trailing which spaces

**Review by enryH — CHANGES_REQUESTED** [review]
at least the trailing white spaces should be removed and the `ttf` file could be renamed:)

File: webinterface/_base.py (hunk 1 above)
Comment (enryH) [inline webinterface/_base.py:38]:
```suggestion
        st.image("webinterface/logos/logo_participants/logos_all_20230926.png")
```

Depending what the root folder the application should be started from, I would suspect that this is works as well. 

I just saw now: you will need to change the link once it is merged as it points to the `Add-logos` branch. So try to make them all relative paths:)

Comment (enryH) [inline webinterface/_base.py:38]:
```suggestion
        st.image("webinterface/logos/logo_participants/logos_all_20230926.png")
```

Depending what the root folder the application should be started from, I would suspect that this is works as well. 

I just saw now: you will need to change the link once it is merged as it points to the `Add-logos` branch. So try to make them all relative paths:)

**Comment by enryH** [comment]
image

What should be above the DDSA logo?

**Comment by mlocardpaulet** [comment]
> in principle only the main logo is needed for the application. So we would not need to provide all of the single logos in each copy of the repository.

Well, our main logos are ProteoBench, EuBIC-MS and EuPA. For the moment, these should be visible when people use the platform and when we make demos. So I think that this is good to have them on the side of the Module.

**Comment by mlocardpaulet** [comment]
> image
> 
> What should be above the DDSA logo?

The ProteoBench logo

**Comment by enryH** [comment]
image

So we need the for 4-5 logs in the side bar and the composite logo?

**Comment by mlocardpaulet** [comment]
> image
> 
> So we need the for 4-5 logs in the side bar and the composite logo?

Oups! I really did not think of dark theme... Is it OK for you if I create a single image with white background for the side-bar logos? Maybe I will remove the DDSA logo and keep only the ProteoBench, EuBIC-MS and EuPA.

**Comment by mlocardpaulet** [comment]
@enryH : I have combined the sidebar logos together, and it now has a white background. It is definitely not ideal. All these logos (main page and module page) could look a lot better. But it is not a priority. The priority is to have the logos present somewhere when we do demos.

**Comment by RobbinBouwmeester** [comment]
@mlocardpaulet, so this can ben merged for now?

**Comment by mlocardpaulet** [comment]
I'd say yes.

**Review by RobbinBouwmeester — APPROVED** [review]

**Pull request merged**